PowerShellEditorServices.sln

```diff
@@ -1,99 +1,99 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Weird, is there a line ending conversion happening that makes these files seem like every line changed?  What response do you get from this command: `git config core.autocrlf`